mobile/src/Student/screens/Game/PhaseTwoBasicGamePlay/index.js

```diff
@@ -219,12 +222,15 @@ const PhaseTwoBasicGamePlay = ({
                 >
                     <View style={styles.roundContainerIncorrect}>
                         <Text style={styles.answerText}>{answer.text}</Text>
+                        {index === selectedAnswerIndex &&
+                            <Image source={require("../img/Picked.png")} />}
                     </View>
                     <Text style={styles.reasonsText}>{answer.reason}</Text>
                 </Card>
             ))}
         </>
 
+    wrongAnswers.map((answer) => (console.log(answer.isChosen)))
```

Comment:
nit: Is the `console.log` needed? If yes, this should be in a separate line and also there's an extra `()` around the statement

Reply:
I think this console.log was added to try to debug the issue with SelectedAnswerIndex we're having on PhaseTwoBasicGamePlay. In the interest of getting this merged, I've removed the console log for now

Reply:
This was for testing purposes, will take it out!